packages/server/src/sdk/app/tables/migration.ts

```diff
@@ -105,14 +113,17 @@ function getColumnMigrator(
   throw new BadRequestError(`Unknown migration type`)
 }
 
-class SingleUserColumnMigrator implements ColumnMigrator {
+abstract class UserColumnMigrator implements ColumnMigrator {
```

Comment:
The vast majority of the 2 classes that now inherit from this was identical, so it felt natural to use an abstract class here.